metadata-ingestion/src/datahub/ingestion/source/sql/hive.py

```diff
@@ -126,10 +670,45 @@ class HiveConfig(TwoTierSQLAlchemyConfig):
     # defaults
     scheme: str = Field(default="hive", hidden_from_docs=True)
 
+    emit_storage_lineage: bool = Field(
+        default=False,
+        description="Whether to emit storage-to-Hive lineage",
+    )
+    hive_storage_lineage_direction: str = Field(
```

Comment:
This would be to have parity with the `glue_s3_lineage_direction` parameter in the [Glue config](https://datahubproject.io/docs/generated/ingestion/sources/glue#config-details)